src/router/schema.ts

```diff
@@ -36,8 +42,14 @@ export async function compose(req, res) {
 			})
 			.options({ stripUnknown: true })
 	);
+	const { validate } = Joi.attempt(
```

Comment:
I think this is unnesessary in `POST /schema/compose`- it must always validate the schema, because it takes versions of services as inputs and you can never guaranteed that any version of any service is compatible and thus full schema is always valid

Reply:
Makes sense. Will adjust.